package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model
+   href="http://download.ros.org/schema/package_format3.xsd"
+   schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+ <name>geographiclib_cmake_module</name>
+ <version>0.0.0</version>
+ <description>Expose CMake targets from older geographiclib libraries for Ubuntu Jammy</description>
+ <maintainer email="25047695+Ryanf55@users.noreply.github.co">Ryan Friedman</maintainer>
```

Comment:
I don't see an MX record for github.co so I suspect this is a typo.

Reply:
```suggestion
 <maintainer email="25047695+Ryanf55@users.noreply.github.com">Ryan Friedman</maintainer>
```
Yep, great catch.